docs/integration/express.js.md

```diff
@@ -431,3 +431,13 @@ To add the dependency to your package.json simply use:
 ```
 npm install sematext-express-agent@latest --save
 ```
+
+## Troubleshooting
+
+If you are not seeing some or any Express.js metrics, you can create a "diagnostics dump" and contact us via live chat or email. To create the diagnostics dump just run the following:
```

Comment:
...just run the following:

=>

just run the following command in the root directory of your project.